geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/util/BatchMultiScanner.scala

```diff
@@ -98,10 +101,10 @@ class BatchMultiScanner(
           val entry = inQ.poll(5, TimeUnit.MILLISECONDS)
           if (entry != null) {
             val entries = collection.mutable.ListBuffer(entry)
-            inQ.drainTo(entries)
+            inQ.drainTo(entries.asJava)
             val task = executor.submit(new Runnable {
               override def run(): Unit = {
-                val iterator = join.copy(ranges = entries.map(joinFunction)).scan(connector, auths, timeout)
+                val iterator = join.copy(ranges = entries.map(joinFunction)).scan(connector, auths, partitionParallelScans, timeout)
```

Comment:
What is this coming to again?

Reply:
this is just passing the parallel flag back into the query plan scan here: https://github.com/locationtech/geomesa/pull/2860/files#diff-db1291fc355346779b080ab328128d55e40f509c9372aa183b54a889511a2f71R127